Readme.md

```diff
@@ -69,14 +69,33 @@ metadata:
 You can provide default values that will be used to initialize the metadata
 of a new post.
 
-### 6. Contribute!
+### 5. Contribute!
 - let me know how it can be improved in the [github
-  issues](https://github.com/jaredly/hexo-admin/issues)
-- [fork](https://github.com/jaredly/hexo-admin) and pull-request
+  issues](https://github.com/yfsoftcom/hexo-admin-yf/issues)
+- [fork](https://github.com/yfsoftcom/hexo-admin-yf) and pull-request
+
+# Qiniu Support!
+
+Get Your Qiniu Keys:
+
+![upload successful](http://olk3bzfd5.bkt.clouddn.com/pasted-1487403384624.png)
+
+Get Your Qiniu Test Domain And Bucket:
+
+![upload successful](http://olk3bzfd5.bkt.clouddn.com/pasted-1487403438509.png)
+To Support Qiniu, add the qiniu variable to your hexo
+`_config.yml`:
+```
+qiniu:
+  bucket: [your qiniu bucket key here]
+  domain: [your qiniu outlink domain here]
+  ACCESS_KEY: [your qiniu access key here]
+  SECRET_KEY: [your qiniu secret key here]
+```
 
 # Credits
 
-built with ❤ by [Jared Forsyth](http://jaredly.github.io)
-([@jaredforsyth](http://twitter.com/jaredforsyth)) using
+built with ❤ by [Wang Fan](http://blog.yfsoft.biz)
+([@yunplus.io](http://blog.yunplus.io)) using
```

Comment:
@yfsoftcom if you are forking the plugin, it would be considerate for you to attribute the original author (me) and the various contributors who have built it.